### ISSUE TITLE
Too many longitude values when aggregating pixels over MODIS' grid

### ISSUE THREAD

**Issue by NikosAlexandris** [NikosAlexandris/gee_sar_fetcher#4]
Running the example https://github.com/NikosAlexandris/gee_sar_fetcher/blob/f5980225636fc7dc2295a5e3d73dcc5b005f50e0/geesarfetcher/example_compose_s1_grd_data_over_epsg4326.py will retrieve and export the requested composite data (after internally slicing and iterating over subregions of the somewhat large(r) requested area of interest) in a more or less expected grid of dimensions: (height x width) (119, 90).

Running the same example, however asking to `.sample` the final composite image (here: mean of values over the requested area and time period) in the 'SR-ORG:6974' grid (= MODIS' Sinusoidal grid used in GEE) (that is https://github.com/NikosAlexandris/gee_sar_fetcher/blob/f5980225636fc7dc2295a5e3d73dcc5b005f50e0/geesarfetcher/example_compose_s1_grd_data_over_srorg6974.py), derives an unexpectedly large number of longitude values. Only. The returned latitude values are, as expected, way less and fit the number of aggregated pixels at the requested spatial resolution (`scale` in GEE's terminology).

See also: https://gist.github.com/NikosAlexandris/900c4ca1dc3a96776d53b06a3e32d65f

Why is this happening?

**Comment by NikosAlexandris** [comment]
To run the examples,
1. clone this repository
```
git clone https://github.com/NikosAlexandris/gee_sar_fetcher.git
```
2. install the package
```
cd geesarfetcher
pip install .
```
3. run the 1st example
```
python geesarfetcher/example_compose_s1_grd_data_over_epsg4326.py
```
4. run the 2nd example
```
python geesarfetcher/example_compose_s1_grd_data_over_epsg4326.py
```

**Comment by NikosAlexandris** [comment]
Output of 1st example: https://pastebin.com/0SFxuYbM
Output of 2nd example: https://pastebin.com/skRbnk8q

**Comment by NikosAlexandris** [comment]
f2ff1122141c04073778e0e959e5bd1203a58047 and c9c71f8e03c242a54a79a0e868050f57394195ee fix the missing `proj` parameter in fetching a "reduced" Image (in GEE's terminology). However, still the number of longitudes equals the total number of pixels, it seems in the area.
```
❯ ipython -i geesarfetcher/example_compose_s1_grd_data_over_srorg6974.py
Python 3.7.6 (default, Apr 13 2020, 23:26:25)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.19.0 -- An enhanced Interactive Python. Type '?' for help.
Region sliced in 16 subregions.
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 16/16 [00:54<00:00,  3.42s/it]
Generating image of shape (height x width) (250, 60750)
100%|█████████████████████████████████████████████████████████████████████████████████████████████| 60750/60750 [00:08<00:00, 6993.42it/s]
```

In degrees, it all seems to work fine:
```
❯ ipython -i geesarfetcher/example_2.py
Python 3.7.6 (default, Apr 13 2020, 23:26:25)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.19.0 -- An enhanced Interactive Python. Type '?' for help.
Region sliced in 25 subregions.
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 25/25 [01:23<00:00,  3.32s/it]
Generating image of shape (height x width) (250, 336)
100%|█████████████████████████████████████████████████████████████████████████████████████████████| 84000/84000 [00:15<00:00, 5565.60it/s]

```

**Comment by NikosAlexandris** [comment]
Fixed with commits d3b62acc550e4f14c12049fe6c6ab6666580b50b to 13f0316faa3af35405d279a6e006bfd8842c5285.

**Comment by NikosAlexandris** [comment]
Both
```
❯ ipython -i geesarfetcher/example_compose_s1_grd_data_over_epsg4326.py
/usr/lib/python3.7/site-packages/IPython/core/interactiveshell.py:925: UserWarning: Attempting to work in a virtualenv. If you encounter problems, please install IPython inside the virtualenv.
  warn("Attempting to work in a virtualenv. If you encounter problems, please "
Python 3.7.6 (default, Mar 19 2020, 01:29:18)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.5.0 -- An enhanced Interactive Python. Type '?' for help.
Region sliced in 4 subregions.
100%|███████████████████████████████████████████████████████████████████████| 4/4 [00:06<00:00,  1.55s/it]
Generating image of shape (height x width) (90, 119)
100%|████████████████████████████████████████████████████████████| 10710/10710 [00:00<00:00, 12814.58it/s]
```
and
```
❯ ipython -i geesarfetcher/example_compose_s1_grd_data_over_srorg6974.py
/usr/lib/python3.7/site-packages/IPython/core/interactiveshell.py:925: UserWarning: Attempting to work in a virtualenv. If you encounter problems, please install IPython inside the virtualenv.
  warn("Attempting to work in a virtualenv. If you encounter problems, please "
Python 3.7.6 (default, Mar 19 2020, 01:29:18)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.5.0 -- An enhanced Interactive Python. Type '?' for help.
Region sliced in 4 subregions.
100%|███████████████████████████████████████████████████████████████████████| 4/4 [00:07<00:00,  1.85s/it]
Generating image of shape (height x width) (76, 130)
100%|██████████████████████████████████████████████████████████████| 9880/9880 [00:00<00:00, 14810.62it/s]
```
give now _rational_ grids (height x width).
